docs/MANPAGE_SYNTAX.md

```diff
@@ -75,8 +88,8 @@ OPTIONS that are followed by an equal sign include an argument after the equal s
 | *example four*     | Example: Can be combined with **--exit**.                                   |
 | *example five*     | The fifth description                                                       |
 
-The table shows an example for a listing of arguments. The contents in the table should be aligned left. If the content in the table conflicts with this, it can be aligned in a way that supports the understanding of the content. If there is a default argument, it **must** listed as the first entry in the table. The default value is shown as **example one**.
-
+The table shows an example for a listing of arguments. The contents in the table should be aligned left. If the content in the table conflicts with this, it can be aligned in a way that supports the understanding of the content. If there is a default argument, it **must** listed as the first entry in the table.\
```

Comment:
```suggestion
The table shows an example for a listing of arguments. The contents in the table should be aligned left. If the content in the table conflicts with this, it can be aligned to support the understanding of the content. If there is a default argument, it **must** be listed as the first entry in the table.\
```